Amazon.QLDB.Driver/client/Session.cs

```diff
@@ -319,6 +329,13 @@ internal virtual CommitTransactionResult CommitTransaction(string txnId, MemoryS
                 .GetResult();
         }
 
+        internal virtual ExecuteStatementResult ExecuteStatement(
```

Comment:
Why was this added?

Reply:
This is an overloaded function to handle `ValueHolder` objects returned by [IQuery.Parameters](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/object-serialization-idda-jun-4-2021/Amazon.QLDB.Driver/query/Query.cs#L34) property.